### PR TITLE
perf: optimize InternalAutoFreezeMeaninglessBlendShapeProcessor

### PR DESCRIPTION
**Pull request by bdunderscore**
This saves over 250ms on my main avatar.

### CODE REVIEW

File: Editor/Processors/SkinnedMeshes/InternalAutoFreezeMeaninglessBlendShapeProcessor.cs [hunk 1]
```diff
@@ -35,35 +40,63 @@ public override void Process(BuildContext context, MeshInfo2 target)
                 value[vertex.BlendShapeBufferVertexIndex] = true;
             }
 
+            Profiler.BeginSample("buffers loop");
+            
+            var shapeCount = buffers.Sum(x => x.Key.Shapes.Count);
+            
+            NativeArray<bool> isMeaningfulArray = new NativeArray<bool>(shapeCount, Allocator.TempJob);
```
Comment (anatawa12) [inline Editor/Processors/SkinnedMeshes/InternalAutoFreezeMeaninglessBlendShapeProcessor.cs:47]:
```suggestion
            using var isMeaningfulArray = new NativeArray<bool>(shapeCount, Allocator.TempJob);
```

not disposing will leak and create warning / error message 4 frames later so we should dispose with `using var` or using statement

File: CHANGELOG.md [hunk 2]
```diff
@@ -131,6 +131,7 @@ The format is based on [Keep a Changelog].
 - Fix non-VRChat project support `#1310`
 - 'shader' doesn't have a float or range property 'prop' error `#1312`
 - Error if all components are on inactive GameObject `#1318`
+- Improved performance in InternalAutoFreezeMeaninglessBlendShapeProcessor `#1325`
```
Comment (anatawa12) [inline CHANGELOG.md:134]: Can you add PR number to Performance Improvements in Changed section instead of adding new entry?

**Review by anatawa12 — CHANGES_REQUESTED** [review]
I didn't know `NativeDisableContainerSafetyRestriction`.
I thought it's impossible to write same position from multiple jobs.

Thank you.

File: Editor/Processors/SkinnedMeshes/InternalAutoFreezeMeaninglessBlendShapeProcessor.cs (hunk 1 above)
Comment (bdunderscore) [reply Editor/Processors/SkinnedMeshes/InternalAutoFreezeMeaninglessBlendShapeProcessor.cs:47]: oops, forgot about that!

**Comment by anatawa12** [comment]
I'll ignore the commitlint failure for this PR.
Could you check commit message before applying the suggestion in the future PRs, please?

**Review by anatawa12 — APPROVED** [review]
LGTM Thank you!

**Comment by bdunderscore** [comment]
> I'll ignore the commitlint failure for this PR. Could you check commit message before applying the suggestion in the future PRs, please?

Ah, I thought you would squash it on merge. I’ll check that in the future.

**Pull request merged**